src/Execution/Execution.php

```diff
@@ -29,9 +29,9 @@ public function __construct(ExecutionContextBuilder $contextBuilder)
     /**
      * @param Schema        $schema
      * @param DocumentNode  $documentNode
-     * @param null          $rootValue
+     * @param object|array  $rootValue
```

Comment:
Can this be something else?

Reply:
I'm not really sure, but in all the tests, it can only be array or object.